src/data.toml

```diff
@@ -1,17 +1,18 @@
+get_addons = "Bunny offers compatibility with Vendetta (discontinued) & Revenge's(Discontinued) plugins and themes, since Vendetta is what Bunny is based on:\n\n### For plugins:\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\n- Alternatively you can kindly, request for other plugins that you may spot others using in the server that aren't available in the plugin web list.\n### For themes:\n\n- You may install any Vendetta themes you like, preferably more recent ones are recommended as older themes are likely outdated.\n- Revenge themes are also supported, excluding custom fonts support.\n- You can install themes from Enmity as well but theme's colours may be different  depending on your device platforms considering discord uses different strings for both android & iOS.\n\nYou may find a curated list of broken plugins in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
 installation_instructions = "### Android\n\n- **Root** with LSPosed - [BunnyXposed](<https://github.com/pyoncord/BunnyXposed/releases/latest>)\n- **Non-root** - [BunnyManager](<https://github.com/pyoncord/BunnyManager/releases/latest>)\n### iOS\n\n- **Jailed**\n - Pre-patched `.ipa` - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n - AltSource Repo - [Pyoncord App Repo](<https://raw.githubusercontent.com/pyoncord/BunnyTweak/main/app-repo.json>)\n-# Note: TrollStore will display an encryption warning which you can ignore\n- **Jailbroken**\n - Rootful and rootless `.deb` - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n\nPlease keep in mind that this channel is not sideload-, jailbreak nor root support.\nPlease keep questions pertaining to those topics elsewhere."
 enable_experiments = "Head to Settings > Bunny > Activate Discord Experiments"
 account_switcher = "Enable Discord Experiments and look for the 'Mobile Account Switcher' experiment and set it to Treatment 1. Hold your profile in the navigation tab to manage your accounts."
 disable_nmu = "The only way to disable NMU is to use Discord version <= 224 and change the experiment's treatment. Keep in mind that support is only provided for the latest stable release and your version may stop working at some point."
-fix_voice_activity = "The cause of this is unclear. To fix it, please disable 'Advanced Voice Activity' in voice settings."
+fix_voice_activity = "### App doesn't register your voice in VC:\n\n- Check if your mic is on and all permissions are granted to the app.\n- The cause of this is unclear. To fix it, please disable `Advanced Voice Activity` in voice settings.\n- Disable `Auto Sensitivity` & `Automatic Gain Control` located in `settings/Voice` if discord still doesn't register your voice."
```

Comment:
this seems like it was fine before, disabling sensitivity and gain doesn't fix anything more than what Advanced Voice Activity does